Makefile

```diff
@@ -2423,7 +2510,21 @@ endif
 	  $(MAKE) -C otherlibs/$$i install || exit $$?; \
 	done
 ifeq "$(build_ocamldoc)" "true"
-	$(MAKE) -C ocamldoc install
+	$(MKDIR) "$(INSTALL_BINDIR)"
```

Comment:
This line is no longer necessary (since you can't invoke just the installation of ocamldoc on its own anymore)